tests/unit/test_models/test_submodels/test_interface/test_li_plating/test_reversible_plating.py

```diff
@@ -0,0 +1,44 @@
+#
+# Test reversible Li plating submodel
+#
+
+import pybamm
+import tests
+import unittest
+
+
+class TestReversiblePlating(unittest.TestCase):
```

Comment:
To get the coverage of reversible plating to 100% you need to add a couple of tests that cover:

lines 44-45:
```python3
else:
    eta_sei = pybamm.Scalar(0)
```

lines 56-63:
```python3
if (
    "Negative electrode Li plating interfacial current density" in variables
    and "Positive electrode Li plating interfacial current density" in variables
    and "Li plating interfacial current density" not in variables
 ):
    variables.update(
        self._get_standard_whole_cell_interfacial_current_variables(variables)
    )
```

You might need to do the same for the irreversible tests.